jsplot/src/main/java/tech/tablesaw/plotly/api/BubblePlot.java

```diff
@@ -38,22 +41,46 @@ public static Figure create(
   }
 
   public static Figure create(
-      String title, Table table, String xCol, String yCol, String sizeColumn) {
+      String title,
+      Table table,
+      String xCol,
```

Comment:
It seems potentially confusing that there's a parameter `xCol` and `xColumn` and only one is used. It's not clear which takes precedence. Can you remove one of the two parameters? (same with `yCol` / `yColumn`). That would also probably fix the Codacy warning that's being generated

Reply:
I agree with the above. I've kept postponing commenting on this PR because it seems to me adding more and more `create` methods is not a good solution in a way; the other is that this method seems something that could be perhaps `private` and expose to the users something more clean so they don't have to think about the difference between `xCol` and `xColumn`. Might be simple of getting rid of the `table` argument altogether and just send columns.

Reply:
@benmccann @emilianbold I updated PR with above comments. Thanks for review.